src/handlers/TechRecordsListHandler.ts

```diff
@@ -26,9 +26,16 @@ export class TechRecordsListHandler<T extends Vehicle> {
         status,
         searchCriteria
       );
-      techRecordItems = this.formatTechRecordItemsForResponse(techRecordItems);
 
-      return techRecordItems;
+      if (techRecordItems.length === 0) {
+        throw new HTTPError(404, HTTPRESPONSE.RESOURCE_NOT_FOUND);
+      }
+
+      if (searchCriteria === SEARCHCRITERIA.SYSTEM_NUMBER && this.multipleRecordsWithSameSystemNumber(techRecordItems)) {
```

Comment:
Do you need the multipleRecordsWithSameSystemNumber function? Would `techRecordItems.length > 0 `not suffice?

Reply:
This is true, I created it as a util function for the purpose that it might get reused if this was extended to searching by vin, I'll push the change